packages/api/middleware/session.go

```diff
@@ -13,7 +13,7 @@ const Key = "session"
 
 type sessionContextKey struct{}
 
-var Store = NewCookieStore([]byte("session_secret"))
+var Store = NewFilesystemStore("", []byte("session_secret"))
```

Comment:
this wont work since there are multiple api nodes and this makes nodes stateful. lets stick to the cookie store to keep the api stateless